packages/scm-extra/src/browser/history/scm-history-widget.tsx

```diff
@@ -308,11 +308,11 @@ export class ScmHistoryWidget extends ScmNavigableListWidget<ScmHistoryListNode>
                     const repoName = repo ? `${this.labelProvider.getName(new URI(repo.provider.rootUri))}` : '';
 
                     const relPathAndRepo = [relPath, repoName].filter(Boolean).join(' in ');
-                    path = ` for ${relPathAndRepo}`;
+                    path = `${relPathAndRepo}`;
                 }
```

Comment:
Yeah, we have to, you are right.. I can localize this preposition (but it depends on language, i'm not sure that it will be correct in different languages), or we can replace it with character (":" or "/"). What do you think?